test/interactions/keyInteractionTests.ts

```diff
@@ -75,7 +77,7 @@ describe("Interactions", () => {
         svg.remove();
       });
```

Comment:
Do you think it would be a simpler yet sufficient test if we instead just test if "bCallback" is not called if "a" is pressed?  What I mean by this is not to touch the setup for "a" at all.